esp/services/ws_access/ws_accessService.hpp

```diff
@@ -40,6 +40,11 @@ class Cws_accessSoapBindingEx : public Cws_accessSoapBinding
 
     virtual void getNavigationData(IEspContext &context, IPropertyTree & data)
     {
+        if (queryComponentConfig().getPropBool("@api_only"))
```

Comment:
These changes aren't obviously related - should they be a separate jira/PR?

Reply:
This is initially what makes eclservices different from eclwatch.  This turns off the UI and restores the default web service forms.
I expect the two to diverge further over time.
I could have separated the creation of eclservices from the ability to run as an application but thought it would be a good exercise in making sure the feature met the needs of dealing with different flavors of application.